ports/google-cloud-cpp/vcpkg.json

```
@@ -1,6 +1,6 @@
 {
   "name": "google-cloud-cpp",
-  "version-string": "1.23.0",
+  "version-string": "1.24.0",
```

```suggestion
  "version-semver": "1.24.0",
```

Switching to a version schema other than "string" will enable vcpkg to perform `>=` constraints in versioning: https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md

---

Thanks, let's look at changing the schema in a separate PR.  This can be merged as-is.